Documentation for 12.5 features

psatyal opened the pull request:
This commit covers documentation for 12.5 features, including:

- Individual non-working days
- Project activity module
- Nextcloud: link and upload files from OpenProject
- The new archive project permission (open to non-admin roles)
- Default work package views (missing docs)

Remaining:

- Screenshots (after feature freeze)
- iCal export (in dev)

psatyal committed:
First commit of docs for 12.5 features

This commit covers documentation for 12.5 features, including:

- Individual non-working days
- Project activity module
- Nextcloud: link and upload files from OpenProject
- The new archive project permission (open to non-admin roles)
- Default work package views (missing docs)

Remaining:

- Screenshots (after feature freeze)
- iCal export (in dev)

psatyal commented:
@as-op This is of course on-going, but, as always, could you please check the links? Merci !

marcalcobe commented on docs/system-admin-guide/working-days/README.md:
```diff
@@ -17,9 +17,17 @@ To change this setting, navigate to *Administration* → *Working days*.
 
+### Specific non-working days
+
+You can also designate specific dates (such as national or local holidays) as non-working days. Work packages in regular scheduling mode will skip these days at they will not be included when calculating duration.
```
@psatyal do we want to emphasise here also that this non-working days work for all the instance and it is not individual?

psatyal replied:
Hi @marcalcobe. Yes, this is explained in the paragraphe below titled "effect on scheduling":

> As an instance-level setting, any change here will affect the scheduling of all work packages in all projects in that instance. It is currently not possible to define working days at a project-level.

marcalcobe commented on docs/user-guide/activity/README.md:
```diff
@@ -8,39 +8,67 @@ keywords: activity
+- the date and time of the (aggregated) changes
+- a list of attributes that were changed
+
+> IMG: An example of a project attribute update
```
Hi @psatyal,
I suppose we are missing the screenshots here no?

psatyal replied:
Yep, all screenshots will be added after the feature freeze in one go. I've just left placeholders now. We don't have to include them in this PR; will make another one for the screenshots when all the features are implemented.

marcalcobe commented on docs/user-guide/nextcloud-integration/README.md:
```diff
@@ -61,7 +61,71 @@ The following video gives you a short overview of how to use this integration:
+### In OpenProject
+
+In addition to listing files attached to a work package, the **Files** tab now also shows you Nextcloud files that are linked to the current work package. Hovering on any linked file with your mouse will give you options to  **open or download the file, show the containing folder in Nextcloud or remove the link**.
+![Empty_status_files](1_0_00-No_files_linked.png)
```
@psatyal this images are old as they miss the function to upload or link existing in Nextcloud.

psatyal replied:
Noted, thank you! Will include when update screenshots after feature freeze.

marcalcobe commented on docs/user-guide/nextcloud-integration/README.md:
```diff
@@ -61,7 +61,71 @@ The following video gives you a short overview of how to use this integration:
+- select a file from your computer, which will be uploaded to Nextcloud and linked to this work package
+- select an existing file in Nextcloud to link to
+
+#### Upload and link file(s)
```
@psatyal and @Kharonus is this ready? Will the code flag be removed with 12.5?

psatyal replied:
Can we get a confirmation that this will be included in 12.5? @Kharonus

marcalcobe reviewed (changes requested):
Hi @psatyal,

I've added some review comments. The biggest concern I have right now is if the uploading files to Nextcloud from OpenProject will be really released in 12.5 (I tagged @Kharonus for clarification).

MayaBerd committed:
Update README.md

MayaBerd committed:
Correcting a typo

Old :  you can choose to included 
New: you can choose to include

as-op committed:
task(docs): fix links to changed headline anchors

as-op committed:
Merge remote-tracking branch 'origin/docs-updates-for-12.5' into docs-updates-for-12.5

psatyal committed:
Added screenshots for Nextcloud features for 12.5

MayaBerd committed:
Fixing a typo

MayaBerd committed:
small changes

psatyal committed:
Added missing screenshots

Added missing screenshots on these pages I edited for 12.5:

- Activities
- Work package views
- Non-working days

And this page I had not edited for 12.5:

- Agile boards

psatyal commented:
Based to @MayaBerd's feedback, added missing screenshots on these pages I edited for 12.5:

- Activities
- Work package views
- Non-working days

And this page I had not edited for 12.5:

- Agile boards

as-op committed:
Merge branch 'release/12.5' into docs-updates-for-12.5

as-op committed:
Merge branch 'release/12.5' into docs-updates-for-12.5

birthe committed:
Update section about advanced filters

birthe committed:
Merge branch 'docs-updates-for-12.5' of https://github.com/opf/openproject into docs-updates-for-12.5

birthe commented:
Updated the section for advanced filters and renamed it to "Filter for attachment file name and content".
@as-op I have changed a few headlines with my commit. Could you please double-check that we do not have any links to these sections? Thanks a lot

MayaBerd committed:
Removing placeholder text for images

as-op committed:
fix typo in link
